### ISSUE TITLE
Entire Mautic Installation 500 Internal Server Error After Clicking Webview Link in Email

### ISSUE THREAD

**Issue by sdoering** [mautic/mautic#11710]
### Mautic Version

4.4.x series

### PHP version

8.0.25

### What browsers are you seeing the problem on?

Chrome

### What happened?

Similar to the Issue [11650](https://github.com/mautic/mautic/issues/11650) I have a Server Error 500 after testing a web view from an Email Newsletter I created. 

I used an incognito tab to subscribe to the newsletter and opened the DOI email to check it out. I used the web view link to test this as well and afterwards my Mautic Dashboard ended in a 500 error. The log messages reads: 

 `[2022-11-13 17:11:38] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given" at /var/www/virtual/USERNAME_REMOVED/mautic.moorwald.com/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php line 101 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given at /var/www/virtual/croci/mautic.moorwald.com/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php:101)"} {"hostname":"HOSTNAME REMOVED","pid":28948}`
 
 

### How can we reproduce this issue?

Step 1: 

### Relevant log output

```shell
[2022-11-13 17:11:38] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given" at /var/www/virtual/USERNAME_REMOVED/mautic.moorwald.com/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php line 101 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): method_exists(): Argument #1 ($object_or_class) must be of type object|string, null given at /var/www/virtual/croci/mautic.moorwald.com/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php:101)"} {"hostname":"HOSTNAME REMOVED","pid":28948}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by Jeandre-Synergy** [comment]
What worked for me:

- Backup audit_log table (if required to keep history of contact actions)
- Delete contents of audit_log table

**Comment by ghost** [comment]
It randomly appears and the one solution is to clear the cache which is extremely bad thing because it means the manual intervention is needed.

Weird thing is that it is inside `try/catch` and it does not catch that error for some reason.

**Comment by jos0405** [comment]
I can confirm this bug on 4.4.5

**Comment by pstrimbu** [comment]
I commented out files/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php lines 101 to 109 then refreshed the page and it came up.  The 500 error did not return after un-commenting the code.
This worked one time.  I haven't tried to repeat it, but it sounds less invasive than the other solutions.

**Comment by ghost** [comment]
Personally what I did was to change `$item  = $model->getEntity($log['objectId']);` into `$item  = $model->getEntity($log['objectId']) ?: '';` in that file.  https://github.com/mautic/mautic/blob/4.4.4/app/bundles/CoreBundle/EventListener/DashboardSubscriber.php#L100
I don't know if it breaks anything, at least it works and does not crash anymore.

The reason of the crash is `$item` is null and the `method_exists` throws exception when the first argument is null and that exception isn't catched by try/catch

**Comment by pstrimbu** [comment]
That fixed it for me.  Thanks!

**Comment by stevedrobinson** [comment]
I got this error tonight after saving configuration on an instance. I can confirm that @TPapajCin 's solution at least got me back up and running. Like he said, I don't know enough about Mautic or this file to know what side effects this creates, and obviously it would be better to understand why a null is getting generated in the first place.

**Comment by regularguy01** [comment]
I am on Mautic 4.4.5.  I can confirm this bug as well.
Apache 2.4
php 8.0.27  
Thanks

**Comment by regularguy01** [comment]
I also can confirm the fix from [TPapajCin](https://github.com/TPapajCin)
worked for me
Corrected the line #100
hard cleared the var/cache dir
cycled httpd and phpfpm
cleared browser as well
Dashboard presents.

**Comment by southeasterntech** [comment]
@TPapajCin  worked for me too

**Comment by Hollowiman** [comment]
@TPapajCin this fix work's here too ( latest 4.4.6 / nginx )

**Comment by RCheesley** [comment]
Is this still a problem for you @sdoering?

**Comment by TonyBogdanov** [comment]
Seems like this issue was fixed here: https://github.com/mautic/mautic/commit/d154c6c598ccf039b18439e78d6c5ff45aaf3a55, which is part of `4.4.8` (maybe even `4.4.7`).

**Comment by RCheesley** [comment]
Closing as resolved in 4.4.8 thanks @TonyBogdanov
